.github/actions/build-interface/action.yml

```diff
@@ -52,7 +56,7 @@ runs:
       working-directory: ${{ inputs.interfacePath }}
       env:
         HUSKY: '0'
-      run: 'npm ci --omit=optional --no-fund --no-audit'
+      run: 'npm ci --no-fund --no-audit'
 
```

Comment:
I had to make this change to get the install to work with the latest dependencies. I'm not sure why, but some dependency down the chain complains about `nx` missing and I couldn't get it to work without including optional packages.

Reply:
I think the "speed increase" of omitting optional-dependencies is negligible anyway...